packages/dom/src/dom/place-caret-at-vertical-edge.js

```diff
@@ -23,7 +24,7 @@ export default function placeCaretAtVerticalEdge(
 	}
 
 	if ( ! rect || ! container.isContentEditable ) {
-		placeCaretAtHorizontalEdge( container, isReverse );
+		placeCaretAtHorizontalEdge( container, isReverse, false );
```

Comment:
Instead of specifying it here, should we add it as a default value to the `mayUseScroll` argument in the definition of `placeCaretAtHorizontalEdge()`?

Reply:
While I agree it'd be better to give it a default, `placeCaretAtVerticalEdge` also has a `mayUseScroll` argument, but it defaults to `true`. Do you think it still makes sense even though they'll have different defaults in these similar functions? It seems confusing to me.

Reply:
Well, they are separate functions, and `placeCaretAtHorizontalEdge`'s default behavior already hints at `false` default value, just by specifying it as a default value we'll be more explicit about it. So I find it more confusing _without_ the default value  🤷